### PR TITLE
Update to 2.36.0 and add an abi package

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
@ryanvolz, do you know what the windows error here is?

File: recipe/bld_atk.bat [hunk 1]
```diff
@@ -1,6 +1,9 @@
 setlocal EnableDelayedExpansion
 @echo on
 
+set "CXXFLAGS=%CXXFLAGS:-GL%"
+set "CFLAGS=%CFLAGS:-GL%"
```
Comment (ryanvolz) [inline recipe/bld_atk.bat]: This is what's causing the current build failure. I don't know how to do what you want, but this is just setting the CFLAGS to `CFLAGS:-GL`.

**Comment by ryanvolz** [comment]
Ok, so fixing the CFLAGS gets the build back to where it was before the attempt at removing "-GL", so it doesn't seem like that was the problem. The build failure that now happens is also what I'm currently seeing at https://github.com/conda-forge/gdk-pixbuf-feedstock/pull/21. I still don't know what to make of that one yet, but my current suspicion is an upstream package with broken host prefix replacement in a `pkg-config` file since that has caused a lot of recent problems.

File: recipe/bld_atk.bat (hunk 1 above)
Comment (ryanvolz) [reply recipe/bld_atk.bat]: Looks like
```
set "CXXFLAGS=%CXXFLAGS:-GL=%"
set "CFLAGS=%CFLAGS:-GL=%"
```
will do what you want.

**Comment by ryanvolz** [comment]
I've narrowed the Windows build error down to a change in `meson`: `0.53.2` works, `0.54.0` produces this error.

**Comment by ryanvolz** [comment]
> I've narrowed the Windows build error down to a change in `meson`: `0.53.2` works, `0.54.0` produces this error.

This looks like the change that does it: https://github.com/mesonbuild/meson/commit/f66b04b0996eae5cd7b0ad007435d5a51f28b691.

The problem is that the build tries to call `g-ir-scanner` directly, but that fails because it is a python script. Previously, `meson` would set up the build so that it called the python executable to run `g-ir-scanner`. It now seems to prefer to get the entire command for running `g-ir-scanner` from `pkg-config` which just points to the path.

The proper fix seems like it belongs upstream in `meson`, or perhaps in `gobject-introspection`, but the easiest workaround would be to stick to using `meson=0.53.2` for now.

**Comment by pkgw** [comment]
<shakes fist at meson>

Filed conda-forge/meson-feedstock#30 for the latest issue, but this LGTM.

**Pull request merged**